### DISCUSSION TITLE
Vote for Liquid pages support in Edge Dev Tools

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#11394] [Ideas]
https://developercommunity.visualstudio.com/t/Edge-Dev-Tools-Support-Liquid-pages/1694152

Regarding to https://www.youtube.com/watch?v=5G9Jz3x7eyE and https://devblogs.microsoft.com/visualstudio/edge-developer-tools-for-visual-studio-preview/

**Comment by Piedone** [comment]
Wouldn't this be potentially a generic ASP.NET Core-related (i.e. not just Orchard) feature?

I'd like to see this for Razor :).

**Reply by xperiandri** [reply]
Razor already works. Try it

**Reply by Piedone** [reply]
Ah, great!
